libraries/stdlib/js/src/kotlin/collections/ArrayList.kt

```diff
@@ -149,7 +151,7 @@ public actual open class ArrayList<E> internal constructor(private var array: Ar
     }
 
     override fun toArray(): Array<Any?> {
-        return js("[]").slice.call(array)
+        return jsArrayPrototype.slice.call(array)
```

Comment:
It doesn't seem that jsArrayPrototype is now used anywhere except this function. Does it make sense to extract it then to a variable?